profiles/package.mask

```diff
@@ -33,6 +33,26 @@
 
 #--- END OF EXAMPLES ---
 
+# Marco Scardovi <mscardovi@icloud.com> (2023-01-11)
+# It's finally time to drop that masterpiece, as before,
+# if You have a stronger opinion to maintain it feel free
+# to drop that mask. removal on 2023-02-11
+acct-group/gatling
+acct-user/gatling
+app-shells/shish
+dev-libs/dietlibc
+net-misc/ncp
+sys-process/minit
+www-servers/gatling
+
+# Alfredo Tupone <tupone@gentoo.org> (2023-01-10)
```

Comment:
Is this block supposed to be here?

Reply:
> Is this block supposed to be here?

I've rebased with master (because of that) and don't know why it takes up it as is. I don't think it is supposed to be here but I fear to do more damages trying to understand how to remove it